src/types/hr/hr.ts

```diff
@@ -0,0 +1,16 @@
+import { UserEntity } from '../user';
+
+export interface HrEntity {
+  id: string;
+  user: UserEntity;
+  email?: string;
+  fullName: string;
+  company: string;
+  maxReservedStudents: number;
+}
+
+export interface HrCreatedResponse {
```

Comment:
Tu jeśli mogę prosić to przenieś `HRCreatedResponse` do `hr.response.ts` (taka konwencja, w której lepiej się odnaleźć)